docs/visual-basic/programming-guide/language-features/arrays/index.md

```diff
@@ -14,257 +12,311 @@ helpviewer_keywords:
   
- The array `students` in the preceding example contains seven elements. The indexes of the elements range from 0 through 6. Having this array is simpler than declaring seven variables.  
+-   The index of the element represents the grade (index 0 represents kindergarten).
   
- The following illustration shows the array `students`. For each element of the array:  
-  
--   The index of the element represents the grade (index 0 represents kindergarten).  
-  
--   The value that’s contained in the element represents the number of students in that grade.  
+-   The value that’s contained in the element represents the number of students in that grade.
   
  ![Picture of array showing numbers of students](../../../../visual-basic/programming-guide/language-features/arrays/media/arrayexampleschool.gif "ArrayExampleSchool")  
 Elements of the "students" array  
+ 
+The following example contains the Visual Basic code that creates and uses the array:
+
+ [!code-vb[simple-array](../../../../../samples/snippets/visualbasic/programming-guide/language-features/arrays/simple-array.vb)]  
+
+The example does three things:
+
+- It declares a `students` array with seven elements. The number `6` in the array declaration indicates the last index in the array; it is one less than the number of elements in the array.
+- It assigns values to each element in the array. Array elements are accessed by using the array name and including the index of the individual element in parentheses.
+- It lists each value of the array. The example uses a [`For`](../../../language-reference/statements/for-next-statement.md) statement to access each element of the array by its index number.
   
- The following example shows how to refer to the first, second, and last element of the array `students`.  
-  
- [!code-vb[VbVbalrArrays#3](../../../../../samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrArrays/VB/Class1.vb#3)]  
-  
- You can refer to the array as a whole by using just the array variable name without indexes.  
-  
- The array `students` in the preceding example uses one index and is said to be one-dimensional. An array that uses more than one index or subscript is called multidimensional. For more information, see the rest of this topic and [Array Dimensions in Visual Basic](../../../../visual-basic/programming-guide/language-features/arrays/array-dimensions.md).  
-  
-##  <a name="BKMK_CreatingAnArray"></a> Creating an Array  
- You can define the size of an array several ways. You can supply the size when the array is declared, as the following example shows.  
+ The `students` array in the preceding example is a one-dimensional array because it uses one index. An array that uses more than one index or subscript is called multidimensional. For more information, see the rest of this article and [Array Dimensions in Visual Basic](../../../../visual-basic/programming-guide/language-features/arrays/array-dimensions.md).  
```

Comment:
should multidimensional be hyphenated?

Reply:
No, it shouldn't. I was inconsistent, so I've removed the hyphen.